rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -458,7 +446,7 @@ void getNewFederationBtcUTXOs(
 
     @Test
     void getNewFederationBtcUTXOs_calledTwice_returnCachedUtxos() {
-        ActivationConfig.ForBlock activations = ActivationConfigsForTest.only(ConsensusRule.RSKIP123).forBlock(0);
+        ActivationConfig.ForBlock wasabiActivations = ActivationConfigsForTest.wasabi100().forBlock(0);
```

Comment:
why wasabi? we are in mainnet so the storage key is always [this one](https://github.com/rsksmart/rskj/pull/2706/files#diff-f42b277f39730e846e15f899b88e1e7e718280d3ea3d4c4d5b035a5d66385923R455) right?